src/extensions/default/NavigationAndHistory/main.js

```diff
@@ -0,0 +1,464 @@
+/*
+ * Copyright (c) 2015 Adobe Systems Incorporated. All rights reserved.
```

Comment:
@nethip An evergreen format of `2016 - present` has been used lately with the copyrights so one doesn't have to go and change them every year 📆